contrib/inventory/consul_io.py

```diff
@@ -133,7 +133,11 @@
 import re
 import argparse
 import sys
-import ConfigParser
+
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
```

Comment:
`six` is [bundled with Ansible](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_python3.html#bundled-six).